DOCUMENTATION.md

```
@@ -15,18 +16,21 @@ Welcome to our documentation guide. Here are some useful tips you need to know!
+1. **_Inline within the repository's code files_**: We have automated processes to extract this information and place it in our Talawa documentation site [docs.talawa.io](https://docs.talawa.io/).
+1. **_In our `talawa-docs` repository_**: Our [Talawa-Docs](https://github.com/PalisadoesFoundation/talawa-docs) repository contains user edited markdown files that are automatically integrated into our Talawa documentation site [docs.talawa.io](https://docs.talawa.io/) using the [Docusaurus](https://docusaurus.io/) package.
 
 ## How to use Docusaurus
+
 The process in easy:
+
 1. Install `talawa-docs` on your system
-1. Launch docusaurus on your system according to the `talawa-docs`documentation. 
-    - A local version of `docs.talawa.io` should automatically launched in your browser at http://localhost:3000/
+1. Launch docusaurus on your system according to the `talawa-docs`documentation.
+   - A local version of `docs.talawa.io` should automatically launched in your browser at http://localhost:3000/
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)